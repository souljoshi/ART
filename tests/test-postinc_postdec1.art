int main()
{
	int b = 15;
	int[10] a;
	
	b++;
	++b;
	printi(b);
	printc('\n');
	b--;
	--b;
	printi(b);
	printc('\n');

	a[0] = 5;
	a[0]++;
	++a[0];
	printi(a[0]);
	printc('\n');
	a[0]--;
	--a[0];
	printi(a[0]);
	printc('\n');
}