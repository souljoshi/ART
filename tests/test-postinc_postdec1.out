17
15
7
5
